Documentation/writing-principles-and-guidelines/spelling-and-typo3-terminology.rst

```diff
@@ -1,111 +1,1006 @@
+        It is displayed as a nested accordion structure (also supporting tabs)
+        to manage child-records.
+
+    ..  _itemprocessor:
+    -   *ItemProcessor*
+
+        An ItemProcessor is a component in TYPO3 that processes and manipulates
+        individual data elements to be used within the FormEngine.
+
+*   L
+
+    ..  _legacy-installation:
+    -   *Legacy Installation*
+
+        TYPO3 can be operated in one of two modes: "Composer Installation"
+        (using the Composer ecosystem and tooling to setup TYPO3, also referred
+        to as "Composer mode") or "Legacy Installation", in which the TYPO3
+        distribution files are simply maintained as files and folders put onto a
+        server.
+
+    ..  _link-browser:
+    -   *Link Browser*
+
+        The Link Browser is a tool in the TYPO3 backend for creating and
+        managing links and references. It can be accessed when inserting links
+        into content elements and is opened as a popup, allowing to pick pages,
+        records, media files, or enter URLS for all fields configured as "link
+        type", or within plain content edited through the RTE.
+
+    ..  _linkhandler:
+    -   *LinkHandler*
+
+        The LinkHandler is a component in TYPO3 that provides advanced link and
+        reference functionalities. Each kind of Link (like: files, pages,
+        records, mails, telephone, ...) is implemented via the LinkHandler API.
+
+    ..  _linkvalidator:
+    -   *Linkvalidator*
+
+        Linkvalidator is a tool in TYPO3 that checks links and references on the
+        website for validity and identifies broken or invalid links. It operates
+        on the level of content elements and their specific data fields.
+
+    ..  _list-view:
+    -   *List View*
+
+        The :guilabel:`Web -> List` view is a view in the TYPO3 backend used for
+        displaying and managing records in a list format, usually for the
+        File List.
+
+*   M
+
+    ..  _maintenance-mode:
+    -   *Maintenance Mode*
+
+        Maintenance Mode in TYPO3 is used to take the website offline
+        temporarily for updates or maintenance. Only maintainers
+        (administrators) are then able to access the backend and/or frontend.
+
+    ..  _maintenance-tool:
+    -   *Maintenance Tool*
+
+        The Maintenance Tool is a tool in the TYPO3 backend used for performing
+        maintenance tasks and system optimizations. It is part of the "Admin
+        Tools" backend module.
+
+    ..  _makeinstance:
+    -   *makeInstance*
+
+        makeInstance is a method in the TYPO3 PHP API, used for creating
+        instances of classes and objects. It can utilize "Dependency Injection"
+        for service classes.
+
+    ..  _modal:
+    -   *Modal*
+
+        A modal is a dialog or pop-up window in TYPO3 that prompts users to
+        enter or confirm information.
+
+    ..  _module:
+    -   *Module*
+
+        A module is an extendable component in TYPO3 that provides various
```

Comment:
A module extends TYPO3 functionality, but a module itself is not extendable (in most cases)